audio/music/rse/battlefactory.asm

```diff
@@ -17,11 +17,12 @@ Music_BattleFactoryRSE_Ch1:
 	note_type 12, 4, 7
 	rest 12
 	rest 16
+	rest 8
 Music_BattleFactoryRSE_Ch1_loop:
 	duty_cycle 2
 	vibrato 18, 2, 4
-	rest 16
 	note_type 6, 4, 7
+	rest 16
```

Comment:
This wasn't actually a desync, but this first rest of the main loop played at speed 12 on the first iteration and then played at speed 6 in all following iterations, which should produce a warning in Crystal Tracker but doesn't yet. The much less convoluted thing to do is to put the appropriate speed 12 rest before the loop and the appropriate speed 6 rest at the start of the loop. The result in-game is the same though.